<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-TIC5DM4" failures="0" tests="2" name="Test5" time="8.822" errors="0" timestamp="9 Aug 2024 11:43:20 GMT">
  <testcase classname="logout" name="@AfterMethod getResult" time="0.0">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String)&amp;quot; because &amp;quot;ReportGenerator.test&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String)" because "ReportGenerator.test" is null
	at ReportGenerator.getResult(ReportGenerator.java:25)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase classname="logout" name="@AfterMethod getResult" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase classname="logout" name="@AfterClass afterClass" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase classname="logout" name="f" time="0.001"/>
  <testcase classname="logout" name="logOut" time="0.001">
    <skipped/>
  </testcase> <!-- logOut -->
</testsuite> <!-- Test5 -->
